docs/tutorials/pytorch.ipynb

```diff
@@ -0,0 +1,507 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "34f10e9f",
+   "metadata": {
+    "id": "lCqHTGRYBZcz"
+   },
+   "source": [
+    "## Setup\n",
+    "\n",
+    "First, we install TorchGeo and all of its dependencies, including PyTorch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "019092f0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install torchgeo"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4db9f791",
+   "metadata": {
+    "id": "dV0NLHfGBMWl"
+   },
+   "source": [
+    "## Imports\n",
+    "\n",
+    "Next, we import PyTorch, TorchGeo, and any other libraries we need. We also manually set the random seed to ensure the reproducibility of our experiments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d92b0f1",
+   "metadata": {
+    "id": "entire-albania"
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import tempfile\n",
+    "\n",
+    "import kornia.augmentation as K\n",
+    "import torch\n",
+    "from torch import nn, optim\n",
+    "from torch.utils.data import DataLoader\n",
+    "\n",
+    "from torchgeo.datasets import EuroSAT100\n",
+    "from torchgeo.models import ResNet18_Weights, resnet18\n",
+    "\n",
+    "torch.manual_seed(0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d13c2db-e5d4-4d83-846b-a2c32774bb44",
+   "metadata": {},
+   "source": [
+    "## Definitions\n",
+    "\n",
+    "If this is your first introduction to deep learning (DL), a natural question might be \"what _is_ deep learning?\". You may also be curious how it relates to other similar buzz words, including artificial intelligence (AI) and machine learning (ML). We can define these terms as follows:\n",
+    "\n",
+    "* AI: when machines exhibit human intelligence\n",
+    "* ML: when machines learn from example\n",
+    "* DL: when machines learn using neural networks\n",
+    "\n",
+    "In this definition, DL is a subset of ML, and ML is a subset of AI. Some common examples of models and applications of these include:\n",
+    "\n",
+    "* AI: Minimax, A*, Deep Blue, video game AI\n",
+    "* ML: OLS, SVM, $k$-means, spam filtering\n",
+    "* DL: MLP, CNN, ChatGPT, self-driving cars\n",
+    "\n",
+    "In this tutorial, we will specifically focus on deep learning, but many of the same concepts are shared with machine learning."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f26e4b8",
+   "metadata": {
+    "id": "5rLknZxrBEMz"
+   },
+   "source": [
+    "## Datasets\n",
+    "\n",
+    "In order to learn by example, we first need examples. In machine learning, we construct datasets of the form:\n",
+    "\n",
+    "$$\\mathcal{D} = \\left\\{\\left(x^{(i)}, y^{(i)}\\right)\\right\\}_{i=1}^N$$\n",
```

Comment:
These math notations are barely visible when looking at the docs page

Reply:
I don't think it looks too bad, the characters are of similar size to the normal text. During the tutorial, I'll likely zoom in so that everyone can see.